Archiving/app/src/main/java/com/tokbox/android/tutorials/archiving/MainActivity.java

```diff
@@ -180,9 +228,14 @@ protected void onResume() {
 
     @Override
     public void onSessionConnectionDataReady(String apiKey, String sessionId, String token) {
+        Log.i(LOG_TAG, apiKey);
```

Comment:
@hbaqai I think we are using log.d for similar logs in other samples. wdyt? Log.d(LOG_TAG, "ApiKey: "+apiKey + " SessionId: "+ sessionId + " Token: "+token); ?